java/runtime/src/main/java/org/ray/runtime/object/ObjectStore.java

```diff
@@ -68,6 +68,18 @@ public ObjectId put(Object object) {
     return putRaw(serialize(object));
   }
 
+  /**
+   * Serialize and put an object to the object store, with the given object id.
+   *
+   * This method is only used for testing.
+   *
+   * @param object The object to put.
+   * @param objectId Object id.
+   */
+  public void put(Object object, ObjectId objectId) {
```

Comment:
Since you've added a `put` method for testing purposes, can you search and update test code not related to this PR? I believe there are some existing test cases which achieve the same by executing `objectStore.putRaw(objectStore.serialize(object), objectId)`.

Reply:
updated `testPutWithDuplicateId`